components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/listener/IDPMgtAuditLogger.java

```diff
@@ -297,7 +309,8 @@ private String buildData(IdentityProvider identityProvider) {
             for (IdentityProviderProperty property : idpProperties) {
                 data.append(joiner);
                 joiner = ", ";
-                data.append("{").append(property.getName()).append(":").append(property.getValue()).append("}");
+                data.append("{").append(property.getName()).append(":").append(LoggerUtils.isLogMaskingEnable ?
```

Comment:
what sort of properties are available here?

Reply:
properties such as idpIssuerName, jwkUri